lib/pengine/group.c

```diff
@@ -136,11 +136,16 @@ group_header(pcmk__output_t *out, int *rc, const pe_resource_t *rsc,
     }
 
     if (attrs != NULL) {
-        PCMK__OUTPUT_LIST_HEADER(out, FALSE, *rc, "Resource Group: %s (%s)",
-                                 rsc->id, (const char *) attrs->str);
+        PCMK__OUTPUT_LIST_HEADER(out, FALSE, *rc, "Resource Group: %s (%s)%s%s%s",
+                                 rsc->id,
+                                 (const char *) attrs->str, desc ? " (" : "",
+                                 desc ? desc : "", desc ? ")" : "");
         g_string_free(attrs, TRUE);
     } else {
-        PCMK__OUTPUT_LIST_HEADER(out, FALSE, *rc, "Resource Group: %s", rsc->id);
+        PCMK__OUTPUT_LIST_HEADER(out, FALSE, *rc, "Resource Group: %s %s%s%s",
```

Comment:
I think this is probably where the extra space after the group name is being added.  If desc is NULL, the space still gets added but then nothing after it does.